allensdk/brain_observatory/behavior/session_apis/data_io/behavior_nwb_api.py

```diff
@@ -120,7 +121,9 @@ def save(self, session_object):
         return nwbfile
 
     def get_behavior_session_id(self) -> int:
-        return int(self.nwbfile.identifier)
+        if self._behavior_session_id is None:
+            self.get_metadata()
+        return self._behavior_session_id
```

Comment:
There needs to be a more structured way of getting values like this from the NWB. We should create a ticket to capture this work.

No changes requested.